floor/floor/driver/raspberry.py

```diff
@@ -165,7 +165,7 @@ def process_bytes(self, data_bytes):
             self.value_ceiling[position] = max(self.value_ceiling[position], value)
             self.value_floor[position] = min(self.value_floor[position], value)
 
-            new_buf[position] = value
+            new_buf[position] = 1 if value else 0
```

Comment:
I think this will need to change to do some noise filtering on the value, but I'll do that in another request later.